ivy/functional/frontends/tensorflow/metrics.py

```diff
@@ -0,0 +1,15 @@
+import ivy
+
+
+def sparse_categorical_crossentropy(
+    y_true,
+    y_pred,
+    from_logits=False,
+    axis=-1
+):
+    if from_logits:
+        y_pred = ivy.softmax(y_pred)
```

Comment:
You may use the out argument of ivy.softmax to do inplace update of the data on y_pred directly in this case I think.